src/js/apps/patients/sidebar/action-sidebar_app.js

```diff
@@ -44,6 +44,7 @@ export default App.extend(extend({
     const flow = this.action.getFlow();
     if (flow) this.listenTo(flow, 'change:_state', this.setAccess);
     this.listenTo(action, 'change:_owner', this.onChangeOwner);
+    this.listenTo(action, 'destroy', this.stop);
```

Comment:
I think the two action listenTos can be made into one with {}

Reply:
Updated, much cleaner 👍